### ISSUE TITLE
Improve wording in log message when searching all prefabs

### ISSUE THREAD

**Issue by madsbangh** [Mr-sB/MissingScriptsFinder#1]
Thank you for this useful tool :)

On line 83, I have noticed this line `Debug.Log("Missing script prefabs count: " + missingScriptPrefabs.Count);` could be interpreted as "I found n missing scripts".
I think it would be helpful to change the wording more towards "Going to search n prefabs for missing scripts".

Note: I might have misunderstood what the log message is trying to say.

**Comment by Mr-sB** [comment]
You are right. : )

If you pull a request, i will merge it.

 Thanks!

**Comment by madsbangh** [comment]
I was testing around a bit and reading through the code, and I think I actually misunderstood the log message. It looks like the intention is communicated fine.

I did however run into something which confused me a bit. When I run "search in all prefabs" in one of our projects, it seems to find some prefab files with unknown script GUIDs:
![image](https://user-images.githubusercontent.com/2969321/139829954-980d5c6a-6a91-43f1-b4f1-dad2d8fe1a6f.png)

It prints no paths though.

I found out that these were some old script GUIDs referencing UI Graphics scripts. I am not sure if this is because Unity's UI system was moved into a package at some point and some GUIDs were changed.

We had a bunch of old UI prefabs, with these GUIDs present in their files, but when re-saved, they just lose them. So there is actually no missing scripts on them when loaded.

I am closing this issue :)
